backend/src/migrations/1543523539612_init.js

```diff
@@ -0,0 +1,98 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+    pgm.createTable('users', {
+        email: { type: 'varchar(254)', primaryKey: true},
+        first_name: {type: 'varchar(30)'},
+        last_name: {type: 'varchar(150)'},
+        password: {type: 'varchar(128)'},
+        bio: {type: 'varchar(500)'}
+    });
+
+/*****************************************************************************/
+    pgm.createTable('activities', {
+        id: 'id',
+        name: {type: 'varchar(250)', notNull: true, unique: true}
+    });
+
+    pgm.createIndex('activities', 'name');
+
+/*****************************************************************************/
+    pgm.createTable('places', {
+        id: 'id',
+        country: {type: 'varchar(250)', notNull: true},
```

Comment:
I would allow country and city to be null to support online activities

Reply:
We could do another switch to select either online or offline activities, but now its not the priority task.
I think we should leave this one as it is, because this table is a catalog of locations, instead we will change `place` field in `Events` to be optional, so all events with `place = null` are considered online events. And later if we'll have time, we can try to implement google maps in it, and sore coordinates of the places in this table.